src/generator/templates/README.md.njk

```diff
@@ -18,7 +18,7 @@ To learn more, see [Client Libraries Explained](https://cloud.google.com/apis/do
 ## Installation
 
 ```sh
-$ npm install googleapis
```

Comment:
This change is going to generate 180 changes the next time the generator runs.  That may be fine 🤷 Typically when I make a generator change, I like to locally run the generator with `--use-cache`, so all the changes get bundled into one.

Reply:
This method has traditionally made me commit like 500 files... I know I'm doing something wrong, but I'm not quite sure what :)

Reply:
Note to self, don't forget those two middle hyphens:

```
npm run generate -- --use-cache
```